### PR TITLE
updated references of base plugins to netcommon

### PR DESCRIPTION
**Pull request by KB-perByte**
Depends-on: https://github.com/ansible/ansible-zuul-jobs/pull/1463

##### SUMMARY

Minimum required ansible.netcommon version is 2.5.1
Updated base plugin references to ansible.netcommon

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/netcommon_ref_update.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+major_changes:
+  - "Minimum required ansible.netcommon version is 2.5.0."
```
Comment (Qalthos) [inline changelogs/fragments/netcommon_ref_update.yaml:3]:
```suggestion
  - "Minimum required ansible.netcommon version is 2.5.1."
```

File: galaxy.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 authors:
   - Ansible Security Community (ansible-security)
 dependencies:
-  "ansible.netcommon": ">=2.0.0"
+  "ansible.netcommon": ">=2.5.0"
```
Comment (Qalthos) [inline galaxy.yml:5]: This has to be 2.5.1 to work
```suggestion
  "ansible.netcommon": ">=2.5.1"
```

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Review by Qalthos — APPROVED** [review]

**Comment by justjais** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by justjais** [comment]
Changes were merged via https://github.com/ansible-collections/cisco.asa/pull/160. Closing the PR

**Pull request closed without merging**